frontend/src/components/Issue/logic/common.ts

```diff
@@ -189,11 +191,31 @@ export const useCommonLogic = () => {
         earliestAllowedTs: task.earliestAllowedTs,
       };
     });
+    const migrationType = taskList[0].migrationType!;
+
     issueCreate.createContext = {
-      migrationType: taskList[0].migrationType!,
+      migrationType,
       updateSchemaDetailList: detailList,
     };
 
+    // If the database is within VCS project, try to find its latest and done migration history from VCS.
+    // If not found, keep the vcsPushEvent field empty and check in backend.
+    if (migrationType === "BASELINE" && detailList.length === 1) {
```

Comment:
@LiuJi-Jim @boojack OK for this cherrypick, but this is easy to forget because we need to change 2 places for such customized logic, one for validate, and one for actual creation. Any idea to improve this?